tests/test_sync.py

```diff
@@ -1,28 +1,29 @@
 #!/usr/bin/env python
 # coding: utf-8
 
-from __future__ import (
-    absolute_import,
-    division,
-    print_function,
-    unicode_literals,
-)
-
+import base64
 import datetime
+import io
 import logging
 import os
 import sys
+import json
+import hashlib
 import unittest
 import uuid
+from argparse import Namespace
 
 import boto3
+from botocore.vendored import requests
```

Comment:
I thought it was:

```
import stdlib
from stdlib import xyz

import 3rdpartylib
import 3rdpartylib2
from 3rdpartylib import xyz
from 3rdpartylib3 import xyz

import localmodule
from localmodule import xyz
```

at least that's what I've observed.  I could be totally wrong ofc.